### PR TITLE
Feature: Add support for publishing robot path

### PR DESCRIPTION
**Pull request by leandropineda**
### Description

- Added method for publishing robot path a list of tuples `(x, y)`
- Fix `yaw` update on fake robot `example.py`

### CODE REVIEW

File: inorbit_edge/robot.py [hunk 1]
```diff
@@ -441,6 +447,49 @@ def publish_odometry(
...
+                "Path has {} points. Only the first {} points will be used.".format(
+                    len(path_points), ROBOT_PATH_POINTS_LIMIT
+                )
+            )
+
+        # Generate ``PathPoint`` protobuf messages
+        # from the list of path point tuples
+        pb_path_points = []
+        for path_point in path_points[:ROBOT_PATH_POINTS_LIMIT]:
+            pb_path_point = PathPoint()
```
Comment (miguelgarcia) [inline inorbit_edge/robot.py:474]: can this be done in a more pythonic way, using `map` perhaps ?

File: inorbit_edge/robot.py [hunk 2]
```diff
@@ -26,6 +29,9 @@
 MQTT_POSE_TOPIC = "ros/loc/data2"
 MQTT_TOPIC_CUSTOM_DATA = "custom"
 MQTT_TOPIC_ODOMETRY = "ros/odometry/data"
+MQTT_TOPIC_PATH = "ros/loc/path"
+
+ROBOT_PATH_POINTS_LIMIT = 1000
```
Comment (miguelgarcia) [inline inorbit_edge/robot.py:34]: Where does this limit come from ? are we doing the same on the agent ?

**Review by miguelgarcia — APPROVED** [review]
LGTM, left a couple of suggestions/comments

File: inorbit_edge/robot.py (hunk 2 above)
Comment (leandropineda) [reply inorbit_edge/robot.py:34]: I guess we can implement some kind of downsampling, or leave that responsibility to the SDK user.

File: inorbit_edge/robot.py (hunk 1 above)
Comment (leandropineda) [reply inorbit_edge/robot.py:474]: Actually, we can. Positional arguments are not allowed but named are:

```python
>>> PathPoint(1,2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: No positional arguments allowed
>>> PathPoint(x=1,y=2)
x: 1.0
y: 2.0
```

File: inorbit_edge/robot.py (hunk 2 above)
Comment (miguelgarcia) [reply inorbit_edge/robot.py:34]: I just wanted to understand this better, the hard limit seems ok at least for now!

File: inorbit_edge/robot.py [hunk 3]
```diff
@@ -441,6 +447,47 @@ def publish_odometry(
...
+                    len(path_points), ROBOT_PATH_POINTS_LIMIT
+                )
+            )
+
+        # Generate ``PathPoint`` protobuf messages
+        # from the list of path point tuples
+        pb_path_points = [
+            PathPoint(x=path_point[0], y=path_point[1])
+            for path_point in path_points[:ROBOT_PATH_POINTS_LIMIT]
+        ]
```
Comment (miguelgarcia) [inline inorbit_edge/robot.py:475]: I had thought something like `path_points[:ROBOT_PATH_POINTS_LIMIT].map(lambda p: PathPoint(x=p[0], y=p[1])`, but using lists comprehension looks great

**Review by miguelgarcia — APPROVED** [review]
LGTM!

**Pull request merged**